nautobot/utilities/templatetags/helpers.py

```diff
@@ -304,32 +304,27 @@ def percentage(x, y):
 
 @library.filter()
 @register.filter()
-def get_docs(model):
-    """Render and return documentation for the specified model.
+def get_docs_url(model):
+    """Return the documentation URL for the specified model.
+
+    Nautobot Core models have a path like docs/models/{app_label}/{model_name}
+    while plugins will have {app_label}/docs/models/{model_name}.
 
     Args:
         model (models.Model): Instance of a Django model
 
     Returns:
-        str: documentation for the specified model in Markdown format
+        str: URL for the documentation of the object.
 
     Example:
-        >>> get_docs(obj)
-        "some text"
+        >>> get_docs_url(obj)
+        "static/docs/models/dcim/site.html"
     """
-    path = "{}/models/{}/{}.md".format(settings.DOCS_ROOT, model._meta.app_label, model._meta.model_name)
-    try:
-        with open(path, encoding="utf-8") as docfile:
-            content = docfile.read()
-    except FileNotFoundError:
-        return "Unable to load documentation, file not found: {}".format(path)
-    except IOError:
-        return "Unable to load documentation, error reading file: {}".format(path)
-
-    # Render Markdown with the admonition extension
-    content = markdown(content, extensions=["admonition", "fenced_code", "tables"])
-
-    return mark_safe(content)
+    return (
```

Comment:
Any chance we could add a test here to make sure the given static docs file actually exists, and perhaps return None if it doesn't? Then we could add logic in the template to only show the (?) button if there actually is a valid docs_url for the model in question, hiding it for example for plugins that don't have docs yet.

Reply:
Yep, I will make this change in the tests and in the codebase. I will use the other example model.

Reply:
After further research, it is possible for Static Files to not be accessible by the Nautobot server directly, like with Static Storages. Since the static files are meant to be served not from Django, but by another webserver. I think keeping this as simply displaying the expected link, that will be acceptable. Perhaps further improvement will be to add logic in the View, maybe an option to disable the help url for specific objects.

Reply:
So, after further investigation, Django.contrib.staticfiles.finders.find will work perfectly for seeing if the file exists in the static filesystem no matter what storage is set to.